src/components/dashboard/main-content/DashboardMainContent.jsx

```diff
@@ -13,6 +13,9 @@ const DashboardMainContent = () => {
 
   return (
     <CourseEnrollments>
+      {/*
+          The children below will only be rendered if there are no course runs.
```

Comment:
what's with the brackets around this comment? Does that do something I'm not familiar with?

Reply:
Yeah, comments in jsx code need to be in brackets.

Reply:
Otherwise it'll just show up as text in the component that's being rendered.

Reply:
Odd. I figured the `/* */` blocking would prevent that.